src/pages/EntitiesSearch.jsx

```diff
@@ -90,7 +88,12 @@ const config = {
         ? body
         : {
             ...body,
-            query: { bool: { must: [{ term: { published: true } }] } },
+            query: {
+                bool: {
+                    must: [{ term: { published: true } }],
+                    must_not: [{ term: { e_type: "generic" } }],
```

Comment:
This can be removed with the next release of the API. Generic entities will not be in the published index.